### DISCUSSION TITLE
Problems with TS Meteor project.

### DISCUSSION THREAD

**Discussion by DavidSichau** [Akryum/meteor-vite#25] [Q&A]
I tried the package with a large ts project.

If I remove refapp:meteor-typescript from the build the builder is complaining about:

```
   While building the application:
   error: Could not find mainModule for 'web.browser' architecture:
   client/main.ts
   Check the "meteor" section of your package.json file?
   error: Could not find mainModule for 'web.browser.legacy'
   architecture:
   client/main.ts
   Check the "meteor" section of your package.json file?
```

If I add the ts compiler to meteor I get the following error:

```
  packages/refapp:meteor-typescript-compiler/meteor-typescript-compiler.ts:314:13:
   Could not find a valid 'tsconfig.json'.
   at MeteorTypescriptCompilerImpl.createWatcher
```
Would it be possible to have a example showing how a ts project can be used with meteor-vite?

**Comment by diavrank** [comment]
I have an example with TS, Vite 4 and Meteor 2.10 here https://github.com/diavrank/theory-swe/pull/5
